src/components/sections/Newsletter/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ButtonExternalLink } from "@/components/Core";
+import { NEWSLETTER_FORM } from "@/constant";
+
+export const Section = () => (
+  <div className="bg-dsfr-blue-1 py-8">
+    <div className="fr-container">
+      <div className="flex flex-col items-start justify-between">
+        <h2 className="text-[1.375rem] leading-7 font-bold mb-3">
+          Recevoir la newsletter de La Suite
+        </h2>
+        <p className="text-[.875rem] leading-6 mb-6">
+          Restez informés des prochaines avancées de La Suite en recevant la
+          newsletter !
+        </p>
+        <p>
+          <ButtonExternalLink
+            href={NEWSLETTER_FORM}
+            aria-label="S'inscrire à la newsletter"
+          >
+            S&apos;inscrire
+          </ButtonExternalLink>
+        </p>
+      </div>
+    </div>
+  </div>
+);
```

Comment:
Minor: On mobile, text seems now to be centered !

<img width="511" alt="Capture d’écran 2024-03-05 à 15 24 15" src="https://github.com/numerique-gouv/lasuite-landingpage/assets/45729124/bbd05ba7-2099-4f52-ab78-17fa907903de">

Reply:
oh nice catch, woops